.circleci/config.yml

```diff
@@ -63,13 +63,21 @@ jobs:
       - setup_remote_docker
       - aws-cli/setup:
           profile-name: awstester
+      - restore_cache:
+          keys:
+            - dependency-packages-store
       - k8s/install-kubectl:
           # requires 1.14.9 for k8s testing, since it uses log api.
           kubectl-version: v1.14.9
       - run:
           name: Run the integration tests
           command: ./scripts/run-integration-tests.sh
           no_output_timeout: 15m
+      - save_cache:
+          key: dependency-packages-store
```

Comment:
Maybe we should use a checksum on go.mod here? Something like this:
```
      - save_cache:
          key: dependency-packages-store-{{ checksum "go.mod" }}
```
And same in the restore of course. From the docs https://circleci.com/docs/2.0/caching/#managing-caches